rllab/dynamics_randomization/test_dynamics_rand.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+"""
+Benchmark model mutation for dynamics randomization
+"""
+from mujoco_py import load_model_from_xml
+from mujoco_py import MjSim
+from mujoco_py import MjViewer
+from rllab.dynamics_randomization import Variations, VariationMethod
+from rllab.dynamics_randomization import VariationDistribution
+
+import numpy as np
+import os
+import os.path as osp
+import xml.etree.ElementTree as ET
+
+#Execute at the root of rllab
+MUJOCO_PY_PATH = os.getcwd()
+TOSSER_XML = osp.join(MUJOCO_PY_PATH,
+                      "rllab/dynamics_randomization/tosser.xml")
+
+# Load original model text into memory
+tosser = ET.parse(TOSSER_XML)
+
+variations = Variations()
+variations.randomize().\
+        attribute("gear").\
+        at_xpath(".//motor[@name='a1']").\
+        with_method(VariationMethod.COEFFICIENT).\
+        sampled_from(VariationDistribution.UNIFORM).\
+        with_range(0.5, 1.5).\
+        randomize().\
```

Comment:
Generally you split new variations into different lines (e.g. no calling randomize twice in one statement). This is part of why I suggested that there be an `add()` method which is void to finish the statement.

See the [builder pattern](https://en.wikipedia.org/wiki/Builder_pattern#Python)